distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -69,115 +56,293 @@ grant
+    permission java.io.FilePermission "${ddf.home.perm}bin_third_party{/}-", "read,execute";
+    permission java.lang.RuntimePermission "createClassLoader";
+    permission java.io.FilePermission "<<ALL FILES>>", "execute";
+    permission java.security.SecurityPermission "insertProvider";
+}
+
+grant codeBase "file:/org.apache.commons.io" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}system.properties", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.attributes", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}pdp${/}-", "read";
+}
+
+grant codeBase "file:/admin-ui" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}metadata", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}metadata${/}-", "read";
+}
+
+grant codeBase "file:/admin-configuration-configupdater" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}-", "read";
+}
+
+grant codeBase "file:/catalog-core-standardframework" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+    permission java.io.FilePermission "<<ALL FILES>>", "execute";
+}
+
+grant codeBase "file:/catalog-plugin-videothumbnail" {
+    permission java.io.FilePermission "${ddf.home.perm}bin_third_party", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}bin_third_party${/}-", "read,write,execute";
+}
+
+grant codeBase "file:/catalog-transformer-zip" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+}
+
+grant codeBase "file:/sample-soap-endpoint" {
+    permission java.io.FilePermission "<<ALL FILES>>", "read,write,delete";
+    permission java.io.FilePermission "-", "read,write,delete";
+    permission java.io.FilePermission "bundleresource:${/}-", "read,write,delete";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}issuer${/}-", "read,write";
+}
+
+grant codeBase "file:/solr-banana-provisioner" {
+    permission java.util.PropertyPermission "solr.client", "read,write";
+}
+
+grant codeBase "file:/io.netty.common" {
+    permission java.io.FilePermission "${/}proc${/}sys${/}net${/}core${/}somaxconn", "read";
+}
+
+grant codeBase "file:/security-certificate-generator" {
+    permission java.security.SecurityPermission "insertProvider";
+}
+
+grant codeBase "file:/platform-parser-xml" {
+    permission java.lang.RuntimePermission "setContextClassLoader";
+}
 
 //
-// Commons-io includes functionality to monitor directories asynchronously
+// itest specific perms
```

Comment:
❓ Doesn't this particular one allows someone to inject these test libraries at runtime and gain access? Or are you confident that they wouldn't even be able to do that?

Reply:
You'd need access to the OS anyways.

Reply:
❓ Should we still put test specific grants in a different policy file that is only loaded by the itests?  I worry about someone adding a grant here for itests that could also impact the operational system.

Reply:
I'll make a ticket to look into separating into multiple files.